### ISSUE TITLE
Mobile app for Android

### ISSUE THREAD

**Issue by stokito** [conversejs/converse.js#3481]
Is it possible to make the ConverseJS as a mobile app?
I found that Electron based desktop client https://github.com/conversejs/converse-desktop so it should be possible to make similarly for Android https://dev.to/ionic/the-easiest-way-for-web-developers-to-build-mobile-apps-1ih8

**Comment by jcbrand** [comment]
Yes it should be doable, but Converse will first need to support push notifications, otherwise you won't receive any messages while the app is in the background.

https://github.com/conversejs/converse.js/issues/1519

**Comment by sjehuda** [comment]
Movim can be installed as a PWA.

What is the expected fashion in which Converse be available to mobile devices?

**Comment by sjehuda** [comment]
Converse is available for Ubuntu Touch.

**Comment by jcbrand** [comment]
Converse can also be installed as a PWA, but without support for background push notifications you'd always have to open the app to see if you got new messages.

**Comment by sjehuda** [comment]
> Converse can also be installed as a PWA, but without support for
> background push notifications you'd always have to open the app to
> see if you got new messages.
> 

Is this information advertised?

If it is not, yet, then I suggest to inform that this is possible.
Doing so, would increase interest and pergaps also help with this
project.

**Comment by jcbrand** [comment]
I'm primarily a developer and not a marketer, although I respect people who are effective in that domain, and would appreciate any help in that arena.

One reason I would be hesitant right now to advertise this feature is because without push notifications the PWA is not that useful, and there are already existing mobile apps that work pretty well.
